src/Uno.UI/UI/Xaml/Controls/MenuFlyout/MenuFlyout.cs

```diff
@@ -80,31 +82,315 @@ public IList<MenuFlyoutItemBase> Items
+		//	//				(popupGrandChildAsDO.As(&popupGrandChildAsUE));
+		//	//				(popupGrandChildAsUE.Transitions = spTransitionCollection);
+		//	//				(popupGrandChildAsUE.InvalidateMeasure());
+		//	//			}
+		//	//		}
+		//	//	}
+		//	//}
+		//	//else
+		//	{
+		//		popup.ChildTransitions = spTransitionCollection;
+		//	}
+		//}
+
+		void AutoAdjustPlacement(MajorPlacementMode pPlacement)
+		{
+			// UNO TODO
+			// Rect windowRect = default;
+			// (DXamlCore.GetCurrent().GetContentBoundsForElement(GetHandle(), &windowRect));
+		}
+
+		public override void ShowAt(FrameworkElement pPlacementTarget)
+		{
+			m_openWindowed = false;
+			base.ShowAtCore(pPlacementTarget);
+		}
+
+		void ShowAtImpl(UIElement pTargetElement, Point targetPoint)
+		{
+			var targetElement = pTargetElement;
+
+			var showOptions = new FlyoutShowOptions();
+			showOptions.Position = targetPoint;
+
+			try
+			{
+				m_openingWindowedInProgress = true;
+
+				ShowAt(targetElement, showOptions);
+			}
+			finally
+			{
+				m_openingWindowedInProgress = false;
+			}
+		}
+
+		void CacheInputDeviceTypeUsedToOpen(UIElement pTargetElement)
+		{
+			// UNO TODO
+			//CContentRoot* contentRoot = VisualTree.GetContentRootForElement(pTargetElement);
+			//InputDeviceTypeUsedToOpen = contentRoot.GetInputManager().GetLastInputDeviceType();
+		}
+
+		// Callback for ShowAt() from core layer
+		void ShowAtStatic(MenuFlyout pCoreMenuFlyout,
+			UIElement pCoreTarget,
+		   Point point)
+		{
+			Debug.Assert(pCoreMenuFlyout != null);
+			Debug.Assert(pCoreTarget != null);
+
+			DependencyObject menuFlyout = pCoreMenuFlyout;
+
+			DependencyObject target = pCoreTarget;
+
+			pCoreMenuFlyout.ShowAtImpl(target as FrameworkElement, point);
+		}
+
+		void OnProcessKeyboardAcceleratorsImpl(ProcessKeyboardAcceleratorEventArgs pArgs)
+		{
+			if (m_tpItems != null)
+			{
+				var itemCount = m_tpItems.Count;
+				for (int i = 0; i < itemCount; i++)
+				{
+					MenuFlyoutItemBase spItem = m_tpItems[i];
+					(spItem as MenuFlyoutItemBase).TryInvokeKeyboardAccelerator(pArgs);
+				}
+			}
+		}
+
+		IMenu IMenu.ParentMenu
+		{
+			get => m_wrParentMenu?.Target as IMenu;
+			set
+			{
+				m_wrParentMenu = new WeakReference(value);
+
+				// If we have a parent menu, then we want to disable the light-dismiss overlay -
+				// in that circumstance, the parent menu will have a light-dismiss overlay that we'll use instead.
+				IsLightDismissOverlayEnabled = value == null;
+
+				Control presenter = GetPresenter();
+				MenuFlyoutPresenter menuFlyoutPresenter = presenter as MenuFlyoutPresenter;
+
+				menuFlyoutPresenter.IsSubPresenter = value != null;
+			}
+		}
+
+		void IMenu.Close()
+		{
+			Hide();
+		}
 
-			_presenter.Items.AddRange(Items);
+		bool IsWindowedPopup()
+		{
+			return false;
 
-			return _presenter;
+			// UNO TODO
+			// return CPopup.DoesPlatformSupportWindowedPopup(DXamlCore.GetCurrent().GetHandle()) && (FlyoutBase.IsWindowedPopup() || m_openingWindowedInProgress);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5205492)